electionsbot/cogs/election.py

```diff
@@ -134,6 +134,11 @@ def getAllCandidates(self):
         for candidate in self.getAllCandidates():
             await ctx.send(embed=candidate.getEmbed())
 
+    @commands.has_role(ROOT_ROLE_ID)
+    @commands.command(aliases=["voters"])
+    async def voterCount(self, ctx):
+        return await ctx.send(f"{await connectPostgres().fetchval('SELECT COUNT(*) FROM votes')} votes recorded!")
```

Comment:
I believe you have to await the connection to postgres, and then await receiving the value - otherwise it might not work; see other functions

Reply:
Interesting. Pica is probably right here.